api.go

```diff
@@ -18,8 +19,43 @@ type DeparturesResponse struct {
 
 // StatusResponse defines the object returned by the /status endpoint
 type StatusResponse struct {
-	Status         string    `json:"status,omitemty"`
-	LastDataUpdate time.Time `json:"last_data_update"`
+	Status              string    `json:"status,omitemty"`
```

Comment:
you probably mean omitempty
```suggestion
	Status              string    `json:"status,omitempty"`
```

Reply:
The eyes of the tiger ! :tiger:

Reply:
I'm Dragon!